packages/textfield/src/Textfield.ts

```diff
@@ -249,6 +249,8 @@ export class TextfieldBase extends ManageHelpText(
             }
         }
         this.value = this.inputElement.value;
+        this.inputElement.style.height = 'auto';
+        this.inputElement.style.height = `${this.inputElement.scrollHeight}px`;
```

Comment:
Please look closer at making this possible via CSS. Relying on a paint requisite measurement on every keystroke feels like overkill here.

Possibly:
```
white-space: pre-line;
    min-height: max-content;
    overflow: clip;
```

In association with better managed whitespace in the actual `render()` method for the sizing text?